Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -804,66 +802,12 @@ final class AppDelegateViewModelTests: TestCase {
 
   }
 
-  func testOpenPushNotification_WhileInForeground() {
```

Comment:
As far as I could tell we don't really need these tests anymore?

Reply:
Actually maybe we want to keep one of these as a test for the `Notification Opened` and `Opened Notification` events when the app is in the foreground?

Reply:
Hmm good question - let me take another look to see if it makes sense with the new APIs.